ubuntu-advantage.1

```diff
@@ -92,6 +92,11 @@ of:
 whether this service would be available if this machine were attached.
 The possible values are \fIyes\fR or \fIno\fR.
 
+.TP
+.B update-info
+Show updates for packages in the system, including all available UA ESM
+related content.
```

Comment:
```suggestion
Show updates for packages in the system that come from UA ESM sources.
```

@renanrodrigo @lucasmoura @blackboxsw Am I correct that we don't want to show _all_ updates here. We only want to show updates that could come from esm-infra or esm-apps, right?